server/src/main/java/org/apache/druid/client/CachingClusteredClient.java

```diff
@@ -358,8 +358,8 @@ private void computeUncoveredIntervals(TimelineLookup<String, ServerSelector> ti
         // Which is not necessarily an indication that the data doesn't exist or is
```

Comment:
The phrase above "This returns intervals..." is strange. I would say "Record in the response context the intervals..."

Reply:
Updated